TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -43,30 +45,24 @@ actions.cds+=/variable,name=deathmark_kingsbane_condition,value=!talent.kingsban
 actions.cds+=/variable,name=deathmark_condition,value=!stealthed.rogue&buff.slice_and_dice.remains>5&dot.rupture.ticking&buff.envenom.up&!debuff.deathmark.up&variable.deathmark_ma_condition&variable.deathmark_kingsbane_condition
 # Usages for various special-case Trinkets and other Cantrips if applicable
 actions.cds+=/call_action_list,name=items
-## Invoke Externals to Deathmark
-## actions.cds+=/invoke_external_buff,name=power_infusion,if=dot.deathmark.ticking
 # Cast Deathmark if the target will survive long enough
 actions.cds+=/deathmark,if=(variable.deathmark_condition&target.time_to_die>=10)|boss&fight_remains<=20
 # Check for Applicable Shiv usage
-actions.cds+=/call_action_list,name=shiv
+actions.cds+=/call_action_list,name=shiv,strict=1,if=cooldown.shiv.ready
 actions.cds+=/kingsbane,if=(debuff.shiv.up|cooldown.shiv.remains<6)&buff.envenom.up&(cooldown.deathmark.remains>=50|dot.deathmark.ticking)|boss&fight_remains<=15
 # Avoid overcapped energy, use with shiv, or dump charges at the end of a fight
-actions.cds+=/thistle_tea,if=!buff.thistle_tea.up&(((energy.deficit>=100+energy.regen_combined|charges>=3)&debuff.shiv.remains>=4)|spell_targets.fan_of_knives>=4&debuff.shiv.remains>=6)|(buff.thistle_tea.down|energy.deficit>100)&fight_remains<charges*6
+actions.cds+=/thistle_tea,if=!buff.thistle_tea.up&debuff.shiv.remains>=4|spell_targets.fan_of_knives>=4&debuff.shiv.remains>=6|boss&fight_remains<=cooldown.thistle_tea.charges*6
```

Comment:
The previous changes were put in to avoid wasting all one's Thistle Tea at the end of a pack, *but* still use Thistle Tea outside of boss fights.  At the very least, it should probably be:

`!buff.thistle_tea.up&(debuff.shiv.remains>=4|spell_targets.fan_of_knives>=4&debuff.shiv.remains>=6|boss&fight_remains<=cooldown.thistle_tea.charges*6)`

That way, it won't spam Thistle Tea while Thistle Tea is active.

Reply:
agree